services/manifest-repo-export-service/pkg/repository/transformer.go

```diff
@@ -1017,13 +1059,14 @@ func isLatestVersion(state *State, application string, version uint64) (bool, er
 
 // Finds old releases for an application: Checks for the oldest release that is currently deployed on any environment
 // Releases older that the oldest deployed release are eligible for deletion. releaseVersionsLimit
-func findOldApplicationVersions(ctx context.Context, transaction *sql.Tx, state *State, name string) ([]uint64, error) {
+func findOldApplicationVersions(ctx context.Context, transaction *sql.Tx, state *State, versionLimit int, appName string) ([]uint64, error) {
```

Comment:
I didn't understand why we added `versionLimit` to the inputs of the function. Because it's always equal to `int(state.ReleaseVersionsLimit)` and we already have `state` as an input